bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdf_pyz.py

```diff
@@ -0,0 +1,277 @@
+import re
+from .._numbadeclare import _NumbaDeclareDecorator
+
+# This class is still to undergo some restructuring thus the docstring may not be complete.
+# The class will be fully realized after the Define has been Pythonized.
+class FunctionJitter:
+    """
+    Class to jit function using NumbaDeclare.
```

Comment:
I'd try to adhere to PEP-8 in all files and stick to max 79 characters per line.